### PR TITLE
chore: mark maintenance tasks migrations safe

### PR DESCRIPTION
**Pull request by VaibhavUpreti**
#### Describe the changes you have made in this PR -

Mark migrations introduced by maintenance_tasks gem as safe.

### CODE REVIEW

**Comment by VaibhavUpreti** [comment]
We will get a true exit code if the container already exists, otherwise kamal will start it.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -42,8 +42,8 @@ jobs:
       - name: Deploy with kamal
         run: |
           kamal lock release
-          kamal accessory boot yosys2digitaljs-server
-          kamal traefik boot
+          kamal accessory boot yosys2digitaljs-server || true
```
Comment (tachyons) [inline .github/workflows/deploy.yml:45]: Move this to separate PR

**Review by tachyons — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (VaibhavUpreti) [reply .github/workflows/deploy.yml]: Done @tachyons, first need to merge this then the other one https://github.com/CircuitVerse/CircuitVerse/pull/4001
